### PR TITLE
[FLINK-19213][docs-zh] Update the Chinese documentation

### PR DESCRIPTION
**Pull request by xiaoHoly**
## What is the purpose of the change

I will translate document avro-confluent.md under formats into document avro-confluent.zh.md.
We should update the Chinese documentation with the changes introduced in FLINK-18802

## Brief change log

-translate  flink/docs/dev/table/connectors/formats/avro-confluent.zh.md

## Verifying this change

This change is a trivial rework / code cleanup without any test coverage.

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): (no)
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: (no)
  - The serializers: (no)
  - The runtime per-record code paths (performance sensitive): (no)
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Kubernetes/Yarn/Mesos, ZooKeeper: (no)
  - The S3 file system connector: (no)

## Documentation

  - Does this pull request introduce a new feature? (no)
  - If yes, how is the feature documented? (no)

### CODE REVIEW

**Comment by xiaoHoly** [comment]
this week ,i will translate this doc. please help me to review and merge

**Comment by xiaoHoly** [comment]
@dawidwys ,hi,Dawid Wysakowicz. I have finished my translation. Can you help me review my code?

**Comment by dawidwys** [comment]
Unfortunately I don't know Chinese.

Maybe @wuchong or @klion26 could help with it, or ping someone who can.

**Comment by xiaoHoly** [comment]
@wuchong hi,Jark,I have finished my translation. Can you help me review my code? It may take you some time. Thank you

**Comment by xiaoHoly** [comment]
> Unfortunately I don't know Chinese.
> 
> Maybe @wuchong or @klion26 could help with it, or ping someone who can.

Well, thank you very much.

**Comment by xiaoHoly** [comment]
> * `@flinkbot run travis` re-run the last Travis build

@flinkbot run azure re-run the last Azure build

**Comment by xiaoHoly** [comment]
> * `@flinkbot run travis` re-run the last Travis build
@flinkbot run travis re-run the last Travis build

File: docs/dev/table/connectors/formats/avro-confluent.zh.md [hunk 1]
```diff
@@ -72,50 +72,50 @@ CREATE TABLE user_behavior (
...
 </table>
 
-Data Type Mapping
+数据类型映射
 ----------------
 
-Currently, Apache Flink always uses the table schema to derive the Avro reader schema during deserialization and Avro writer schema during serialization. Explicitly defining an Avro schema is not supported yet.
-See the [Apache Avro Format]({% link dev/table/connectors/formats/avro.zh.md%}#data-type-mapping) for the mapping between Avro and Flink DataTypes. 
+目前 Apache Flink 都是从 table schema 去推断反序列化期间的 Avro reader schema 和序列化期间的 Avro writer schema。显式地定义 Avro schema 暂不支持。
+[Apache Avro Format]({% link dev/table/connectors/formats/avro.zh.md%}#data-type-mapping)中描述了flink数据和Avro数据的对应关系。
```
Comment (wuchong) [inline docs/dev/table/connectors/formats/avro-confluent.zh.md:117]:
```suggestion
[Apache Avro Format]({% link dev/table/connectors/formats/avro.zh.md%}#data-type-mapping)中描述了 Flink 数据类型和 Avro 类型的对应关系。 
```

File: docs/dev/table/connectors/formats/avro-confluent.zh.md [hunk 2]
```diff
@@ -72,50 +72,50 @@ CREATE TABLE user_behavior (
...
+数据类型映射
 ----------------
 
-Currently, Apache Flink always uses the table schema to derive the Avro reader schema during deserialization and Avro writer schema during serialization. Explicitly defining an Avro schema is not supported yet.
-See the [Apache Avro Format]({% link dev/table/connectors/formats/avro.zh.md%}#data-type-mapping) for the mapping between Avro and Flink DataTypes. 
+目前 Apache Flink 都是从 table schema 去推断反序列化期间的 Avro reader schema 和序列化期间的 Avro writer schema。显式地定义 Avro schema 暂不支持。
+[Apache Avro Format]({% link dev/table/connectors/formats/avro.zh.md%}#data-type-mapping)中描述了flink数据和Avro数据的对应关系。 
 
-In addition to the types listed there, Flink supports reading/writing nullable types. Flink maps nullable types to Avro `union(something, null)`, where `something` is the Avro type converted from Flink type.
+除了此处列出的类型之外，Flink 还支持读取/写入可为空的类型。 Flink 将可为空的类型映射到 Avro `union(something, null)`, 其中 `something` 是从 Flink 类型转换的 Avro 类型。
```
Comment (wuchong) [inline docs/dev/table/connectors/formats/avro-confluent.zh.md:119]:
```suggestion
除了此处列出的类型之外，Flink 还支持读取/写入可为空（nullable）的类型。 Flink 将可为空的类型映射到 Avro `union(something, null)`, 其中 `something` 是从 Flink 类型转换的 Avro 类型。
```

File: docs/dev/table/connectors/formats/avro-confluent.zh.md [hunk 3]
```diff
@@ -29,27 +29,27 @@ under the License.
 * This will be replaced by the TOC
 {:toc}
 
-The Avro Schema Registry (``avro-confluent``) format allows you to read records that were serialized by the ``io.confluent.kafka.serializers.KafkaAvroSerializer`` and to write records that can in turn be read by the ``io.confluent.kafka.serializers.KafkaAvroDeserializer``. 
+Avro Schema Registry (``avro-confluent``) 格式能让你读取被 ``io.confluent.kafka.serializers.KafkaAvroSerializer``序列化的记录并可以写入成 ``io.confluent.kafka.serializers.KafkaAvroDeserializer``反序列化的记录。
```
Comment (wuchong) [inline docs/dev/table/connectors/formats/avro-confluent.zh.md:32]:
```suggestion
Avro Schema Registry (``avro-confluent``) 格式能让你读取被 ``io.confluent.kafka.serializers.KafkaAvroSerializer``序列化的记录，以及可以写入成能被 ``io.confluent.kafka.serializers.KafkaAvroDeserializer``反序列化的记录。
```

File: docs/dev/table/connectors/formats/avro-confluent.zh.md [hunk 4]
```diff
@@ -29,27 +29,27 @@ under the License.
...
 
-When reading (deserializing) a record with this format the Avro writer schema is fetched from the configured Confluent Schema Registry based on the schema version id encoded in the record while the reader schema is inferred from table schema. 
+当以这种格式读取（反序列化）记录时，将根据记录中编码的 schema 版本 id 从配置的 Confluent Schema Registry 中获取 Avro writer schema ，而从 table schema 中推断出 reader schema。
 
-When writing (serializing) a record with this format the Avro schema is inferred from the table schema and used to retrieve a schema id to be encoded with the data. The lookup is performed with in the configured Confluent Schema Registry under the [subject](https://docs.confluent.io/current/schema-registry/index.html#schemas-subjects-and-topics) given in `avro-confluent.schema-registry.subject`.
+当以这种格式写入（序列化）记录时，Avro schema 是从 table schema 中推断出来的，并用于检索要与数据一起编码的 schema id。检索是在 Confluent Schema Registry 配置中的 `avro-confluent.schema-registry.subject` 中指定的[subject](https://docs.confluent.io/current/schema-registry/index.html#schemas-subjects-and-topics)下执行的。
```
Comment (wuchong) [inline docs/dev/table/connectors/formats/avro-confluent.zh.md:36]:
```suggestion
当以这种格式写入（序列化）记录时，Avro schema 是从 table schema 中推断出来的，并会用来检索要与数据一起编码的 schema id。我们会在配置的 Confluent Schema Registry 中，配置的 [subject](https://docs.confluent.io/current/schema-registry/index.html#schemas-subjects-and-topics) 下，检索 schema id。subject 通过 `avro-confluent.schema-registry.subject` 参数来制定。
```

**Review by wuchong — COMMENTED** [review]
Thanks for the translation. I left some comments.

File: docs/dev/table/connectors/formats/avro-confluent.zh.md (hunk 4 above)
Comment (xiaoHoly) [reply docs/dev/table/connectors/formats/avro-confluent.zh.md:36]: thanks,i will update my code with your comments

**Review by wuchong — APPROVED** [review]
LGTM.

**Pull request merged**